Monotype/c2pa-rs
# C2PA-55: Use of the `C2PA` font table

# JIRA Issue

- https://monotype.atlassian.net/browse/C2PA-55

# Checklist


# Notes for Reviewers




Adds support for the `C2PA` font table for storing/reading an embedded manifest store in a font.

Additionally, support for writing a remote manifest URI is added, but there is currently no way to read the value from the font. Also there appears to be plans to change how read/write of remote manifests are done.

# Verification Instructions


Should be able to sign a font and see a new `C2PA` table in the font.

Use the changes in https://github.com/Monotype/c2patool/pull/4 for verifying.

Since the `"otf"` feature is optional and off by default, to run the unit tests added you will need to run one of the following:

- `cargo test --lib asset_handlers::otf_io --features "otf, file_io"` - to run only the `otf_io` unit tests
- `cargo test --all-features` - to run all unit tests, including `otf_io` unit tests.

> Actively maintained by the @Monotype/driverpdldev team.
